extensions/python/ExecutePythonProcessor.h

```diff
@@ -35,10 +35,6 @@
 #include "PythonScriptEngine.h"
 #include "PythonScriptEngine.h"
 
-#if defined(__GNUC__) || defined(__GNUG__)
-#pragma GCC visibility push(hidden)
```

Comment:
This was added due to pybind11 requiring -fvisibility=hidden, but as we do not use pybind11 anymore this can be removed